### PR TITLE
Tidy the changelog a bit

### PR DESCRIPTION
**Pull request by henriquegemignani**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [6.3.0] - 2023-10-??
+
+- Nothing yet.
+
+## [6.2.1] - 2023-09-??
+
+- Nothing.
```
[the remark this answers is not in the document]
Comment (henriquegemignani) [reply CHANGELOG.md:14]: Just me being fancy. 6.2.1 might not exist, but 6.3.0 certainly will.

[the remark this answers is not in the document]
Comment (Miepee) [reply CHANGELOG.md:14]: "Not yet determined" feels nicer for that

[the remark this answers is not in the document]
Comment (henriquegemignani) [reply CHANGELOG.md:14]: Do you want me to change and cause CI to restart just for that?

[the remark this answers is not in the document]
Comment (Miepee) [reply CHANGELOG.md:14]: Depends. Is that specific part of the changelog visible anywhere else except viewing the readme file manually? If no, then no.

[the remark this answers is not in the document]
Comment (henriquegemignani) [reply CHANGELOG.md:14]: The CHANGELOG.md file is only ever used when you check it yourself.
The in-app release notes uses the GitHub Releases, which I copy paste manually there.

**Review by Miepee — APPROVED** [review]

**Pull request merged**